components/light/index.rst

```diff
@@ -382,11 +384,12 @@ that the light is completely OFF, and ON means that the light is emitting at lea
 
 .. _light-on_turn_on_off_trigger:
 
-``light.on_turn_on`` / ``light.on_turn_off`` Trigger
-****************************************************
+``light.on_turn_on`` / ``light.on_turn_off`` / ``light.on_state`` Trigger
```

Comment:
Maybe make a new heading for `on_state`, it seems weird the way the paragraph works.

Reply:
Makes sense. I split it off into its own header now.